src/utils/apiRequests/validateToken.ts

```diff
@@ -0,0 +1,14 @@
+import { decodeToken } from './decodeToken';
+
+export const validateToken = (token: string): boolean => {
+  try {
+    const decoded = decodeToken(token);
```

Comment:
If `decodeToken` is only used here, it does not help the readability of the the code to extract it into an own file rather than just add it as a function directly in this module.

Reply:
done